data_models/preprocessed_contents.py

```diff
@@ -0,0 +1,43 @@
+''' Class definition for Pre processed contents'''
+
+
+class PreprocessedContents:
+    def __init__(
+            self,
+            title_text,
+            normal_text,
+            media,
+            embedded_content,
+            quoted_content):
+        self.title_text = title_text
+        self.normal_text = normal_text
+        self.media = media
+        self.embedded_content = embedded_content
+        self.quoted_content = quoted_content
+
+        self._calculate_content_counts()
+
+    def _calculate_content_counts(self):
+        # calculate all counts once so we
+        # don't have to recalculate everytime
+        # it's needed
+        self.title_text_content_count = len(self.title_text)
+        self.normal_text_content_count = len(self.normal_text)
+        self.media_content_count = len(self.media)
+        self.embedded_content_count = len(self.embedded_content)
+        self.quoted_content_count = len(self.quoted_content)
```

Comment:
Can you run pylint for this file?
You still don't need such variables. You can direclty run len(self.title_text)